internal/db/blockmap_test.go

```diff
@@ -20,11 +21,20 @@ func genBlocks(n int) []protocol.BlockInfo {
 	b := make([]protocol.BlockInfo, n)
 	for i := range b {
 		h := make([]byte, 32)
-		for j := range h {
-			h[j] = byte(i + j)
+		for i := 0; i < len(h)/4; i += 4 {
```

Comment:
This is all to generate blocks with actual random hashes, instead of just a few hashes repeated.